lib/teracy-dev-core/processors/variables.rb

```diff
@@ -6,7 +6,14 @@ module Processors
     class Variables < TeracyDev::Processors::Processor
 
       def process(settings)
-        @logger.debug("process: #{settings}")
+        @logger.debug("process: #{settings['variables']}")
+
+        # see `settings['variables']` as a template, then load it using `envsubst`
+        variables = `envsubst <<< "#{settings['variables']}"`
```

Comment:
note: you can not assume `envsubst` is always available on macOS, Linux, Windows.